tests/System.Text.Formatting.Tests/CustomTypeFormatting.cs

```diff
@@ -25,8 +25,14 @@ public bool TryFormat(Span<byte> buffer, out int bytesWritten, TextFormat format
 
 
             char symbol = _inMonths ? 'm' : 'y';
+            int consumed;
             int symbolBytes;
-            if (!encoding.TextEncoder.TryEncode(symbol, buffer.Slice(bytesWritten), out symbolBytes)) return false;
+
+            unsafe
+            {
+                ReadOnlySpan<char> symbolSpan = new ReadOnlySpan<char>(&symbol, 1);
+                if (!encoding.TextEncoder.TryEncode(symbolSpan, buffer.Slice(bytesWritten), out consumed, out symbolBytes)) return false;
```

Comment:
Wrap return false in curly brackets? Looking at other areas in this PR, for consistency.